tokens/blocket.se/background.yml

```
@@ -4,4 +4,8 @@ token: maps
 color:
   background:
     _: white
-    subtle: bluegray-50
\ No newline at end of file
+    subtle: bluegray-50
+    interactive: white
+    interactive:
```

Shouldn't this be like: 🤔 
```suggestion
    interactive:
      _: white
```

---

Ah ofc, tnx I fix